pom.client.xml

```diff
@@ -408,6 +408,87 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-enforcer-plugin;external_dependency} -->
+        <executions>
+          <execution>
+            <id>default-cli</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <!--
+                Rule: https://maven.apache.org/enforcer/enforcer-rules/bannedDependencies.html
+                We have a list of blessed dependencies here: https://azure.github.io/azure-sdk/java_implementation.html#dependencies
+                Format of includes / excludes is the following: 'groupId[:artifactId][:version][:type][:scope][:classifier]'.
+                Wildcards can be used.
+                -->
+                <bannedDependencies>
+                  <searchTransitive>false</searchTransitive>
+                  <excludes>
+                    <!-- We exclude everything that is runtime-time, compile-time, or provided scoped (i.e. we allow
+                    dependencies when they are used for test scope or otherwise) -->
+                    <exclude>*:*:*:*:runtime</exclude>
+                    <exclude>*:*:*:*:compile</exclude>
+                    <exclude>*:*:*:*:provided</exclude>
+                  </excludes>
+                  <includes>
+                    <include>com.azure:*</include>
+                    <include>io.projectreactor</include>
+                    <include>io.projectreactor.netty</include>
+                    <include>io.netty</include>
+                    <include>com.squareup.okhttp3</include>
+                    <include>org.slf4j</include>
+                    <include>com.fasterxml.jackson.*</include>
+                    <include>io.opentelemetry</include>
+                    <include>net.java.dev.jna</include>
```

Comment:
Mind adding a comment for what this is and where it is being used.

Reply:
I've moved it into the azure-identity whitelist